safecast_test.go

```diff
@@ -255,3 +257,209 @@ func ExampleMustRound() {
 	fmt.Println("not reached", out) // not reached
 	// Output: panic: safecast: out of range for -128.6 (float64) to int8
 }
+
+// We need this function, because 'f' format does not work as expected.
+// Consider the following code:
+//
+//	src := 72057594037927936 // 2^56
+//	dst := safecast.MustConvert[float32](src)
+//	t.Logf("0x%x -> %s (%s)", src,
```

Comment:
%f works perfectly fine (as I use in the existing t.Errorf)

https://go.dev/play/p/WlqUe0EolO8

Reply:
and the doc explains what -1 does... it's the "fewest digits that yields the same number" not the best representation possible

https://go.dev/play/p/uEo7IqIiPA4